development-docs/TESTING.md

```diff
@@ -224,6 +224,11 @@ If you want to use your own images with a different tag or from a different repo
 
 To set custom Kafka version in system tests you need to set the environment variable `ST_KAFKA_VERSION` to one of the values in [kafka-versions](kafka-versions.yaml).
 
+#### Using private registries
+
+If you want use private registries, before executing the tests you have to create secret and then specify name of the created secret in env variable called
+`STRIMZI_IMAGE_PULL_SECRET` with the container registry credentials to be able pull images. Note that secret has to be created in `default` namespace.
```

Comment:
I wonder if the name is confusing given the existence of `STRIMZI_IMAGE_PULL_SECRETS` in Cluster Operator.

Reply:
I have changed it to -> `SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET` as @Frawless suggested.